packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -39,27 +39,12 @@ export const RemoteVideoTile = React.memo(
       showMuteIndicator
     } = props;
 
-    useEffect(() => {
-      if (isAvailable && !renderElement) {
-        onCreateRemoteStreamView && onCreateRemoteStreamView(userId, remoteVideoViewOption);
-      }
-      if (!isAvailable) {
-        onDisposeRemoteStreamView && onDisposeRemoteStreamView(userId);
-      }
-    }, [
-      isAvailable,
-      onCreateRemoteStreamView,
-      onDisposeRemoteStreamView,
-      remoteVideoViewOption,
-      renderElement,
-      userId
-    ]);
-
-    useEffect(() => {
-      return () => {
-        onDisposeRemoteStreamView && onDisposeRemoteStreamView(userId);
-      };
-    }, [onDisposeRemoteStreamView, userId]);
+    if (isAvailable && !renderElement) {
+      onCreateRemoteStreamView && onCreateRemoteStreamView(userId, remoteVideoViewOption);
```

Comment:
note: is there a chance this will be called twice, when isAvailable but before the renderElement has been created? (not sure how long `onCreateRemoteStreamView` takes for renderElement to become defined

Reply:
Yes, that is possible. I think onCreateRemoteStreamView calls an SDK method that is idempotent but I could be wrong.